packages/client/components/ReflectionCard/ReflectionCard.tsx

```diff
@@ -254,12 +319,31 @@ const ReflectionCard = (props: Props) => {
     !isComplete &&
     !isDemoRoute() &&
     (isHovering || !isDesktop)
+
+  const isGroupPhase = !isComplete && phaseType === 'group'
+  const hasNoGroup = !reflectionGroups.some((group) => group.reflections.length > 1)
+  const isMeetingNumberLessThanThree = meetingNumber < 3
+
+  const shouldAnimate = (() => {
+    if (isGroupPhase && hasNoGroup && isMeetingNumberLessThanThree) {
+      const firstReflection = reflectionGroups?.[0]?.reflections?.[0]
```

Comment:
Yes, I am not able to figure out to resolve that.
I already mentioned it here: https://github.com/ParabolInc/parabol/pull/6910#issuecomment-1192243055

Apart from that, here is one more issue: https://github.com/ParabolInc/parabol/pull/6910#issuecomment-1193069249

Reply:
The solution would be to also grab the `reflectPrompts`, take the first `id` and search the first reflection to match the `promptId`. You can take a look [here](https://github.com/ParabolInc/parabol/blob/85d92aa5eba8b4f069b2dbf32836e09805ca60f1/packages/client/components/GroupingKanban.tsx#L80) how the reflections are sorted in their columns.

Reply:
I think the best solution would be to use existing information instead and pass down `isFirstColumn` from `GroupingKanban` and then `isFirstGroup` from `GroupingKanbanColumn`. This would also avoid the additional render caused by the additional GraphQL fields.